pipeline/azure-pipelines.yml

```diff
@@ -0,0 +1,244 @@
+variables:
+- group: sonarsource-build-variables
+- name: BuildParameters.solution
+  value: SonarLint.VisualStudio.Integration.sln
+
+name: $(Build.BuildId)
+
+trigger:
+  branches:
+    include:
+    - master
+    - branch-*
+  batch: True
+
+jobs:
+- job: Phase_1
+  strategy:
+    maxParallel: 2
+    matrix:
+      2019:
+        vsTargetVersion: 2019
+      2022:
+        vsTargetVersion: 2022      
+  displayName: "Build:"
+  cancelTimeoutInMinutes: 1
+  pool:
+    vmImage: windows-2022
+    
+  steps:
+  - checkout: self
+    submodules: recursive
+    fetchTags: false
+    persistCredentials: True
```

Comment:
Why do we need to set `persistCredentials` to true? (looks like the [default ](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-checkout?view=azure-pipelines)is false).

Reply:
This was the result of the export.